core/cmd/local_client.go

```diff
@@ -105,31 +105,51 @@ func loggedStop(app chainlink.Application) {
 }
 
 func checkFilePermissions(rootDir string) error {
-	errorMsg := "%s has overly permissive file permissions, should be atleast %s"
-	keysDir := filepath.Join(rootDir, "tempkeys")
-	protectedFiles := []string{"secret", "cookie"}
-	err := filepath.Walk(keysDir,
-		func(path string, info os.FileInfo, err error) error {
+	// Ensure `$CLROOT/tls` directory (and children) permissions are <= `ownerPermsMask``
```

Comment:
Should we be checking who owns the file?

Reply:
we can't be 100% sure now who creates the file. In a docker setup that would currently be root, but if you run the executable under your own specified user we wouldn't know about that right now

Reply:
Sorry, I meant checking that the file is owned by the same user that is running the CL node